package.json

```diff
@@ -50,7 +50,7 @@
   "engines": {
     "npm": "please-use-pnpm",
     "pnpm": "8",
-    "node": "^20"
+    "node": ">= 20"
```

Comment:
Why enable the version that is greater than 20. Should we keep the same version for every developer?

Reply:
I am using node 21, but agree with you 🖤